### ISSUE TITLE
IthrottleIDTable  to be added as part of online_fpc.yml

### ISSUE THREAD

**Issue by d200201123** [Juniper/healthbot-rules#179]
linecard.statistics/check-ithrottle-statistics uses IthrottleIDTable in ithrottle.yml.  
"target" need to be dynamically learned , hence the table need to be added under online_fpc.yml but it's currently missing.

```
IthrottleIDTable:
  command: show ithrottle id {{ id }}
  args:
    id: 0
  item: '*'
  target: Null
  view: IthrottleIDView

IthrottleIDView:
  regex:
    min-usage: 'Min Usage Perc:    (\d+\.\d+)'
    max-usage: 'Max Usage Perc:    (\d+\.\d+)'
    usage-enable: 'AdjustUsageEnable: (\d)'
  fields:
    throttle-stats: _ThrottleStatsTable

_ThrottleStatsTable:
  title: Throttle Stats
  delimiter: ":"
```

Please update the online_fpc.yml file.

**Comment by gvarao** [comment]
Addressed through #178

**Comment by gvarao** [comment]
Code merged https://github.com/Juniper/healthbot-rules/pull/178
